src/setup/rgw_setup

```diff
@@ -34,6 +37,21 @@ class SetupCmdBase(Cmd):
         parser.add_argument('--services', default='services', help='services')
         parser.add_argument('--config', default='config_url', help='config')
 
+    def _load_config(self, config_url: str):
+        """Load Config url."""
+        self.conf = MappedConf(config_url)
```

Comment:
We can merge this into the __init__ itself 
```
  def __init__(self, *args):
    super().__init__(*args)
    self._conf = MappedConf(config_url)
```

Reply:
done.